src/types/types.ts

```diff
@@ -793,6 +824,217 @@ export interface ServiceAccountInput {
   groups?: Groups;
 }
 
+export interface Model3DListRequest {
+  limit?: number;
+  /**
+   * Filter based on whether or not it has published revisions.
+   */
+  published?: boolean;
+}
+
+export interface CreateRevision3D {
+  /**
+   * True if the revision is marked as published.
+   */
+  published?: boolean;
+  /**
+   * Global rotation to be applied to the entire model.
+   * The rotation is expressed by Euler angles in radians and in XYZ order.
+   */
+  rotation?: [boolean, boolean, boolean];
+  camera?: RevisionCameraProperties;
+  /**
+   * The file id to a file uploaded to Cognite's Files API.
+   * Can only be set on revision creation, and can never be updated. _Only FBX files are supported_.
+   */
+  fileId: CogniteInternalId;
+}
+
+export type Tuple3<T> = [T, T, T];
+
+export interface UpdateRevision3D {
+  id: CogniteInternalId;
+  update?: {
```

Comment:
Shouldn't this be required?

Reply:
at least not in spec...

Reply:
Then that is wrong. It should be required. It doesn't make any sense to update something without providing this prop

Reply:
agree, will fix here